README.md

```diff
@@ -462,13 +460,62 @@ Additionally, the OIDC `redirect_uri` parameter must be set to the HTTP API's au
 This is required, as the Authorization Code Grant needs to be redirected back to the HTTP API, to complete
 HTTP API token generation.
 
-### Add your specified `irods_user_claim` to the user's claims
+### Configure your 'user_mapping'
 
-Currently, the server looks for the custom claim in the ID Token, which is specified in the `irods_user_claim` parameter.
-This serves as the mapping mechanism for an OIDC User to an iRODS User.
+Before you can use OpenID with iRODS, you must enable one of the two shipped plugins, or develop your own to suit your specific needs.
 
-A user who authenticates but does not have the claim specified in `irods_user_claim` mapped in their account
-will not have access to the API. A HTTP 400 Bad Request status code will be returned if the claim specified in `irods_user_claim` is not found.
+#### Local File plugin
+
+To use this plugin, make sure to load `libirods_http_plugin-local_file.so`.
```

Comment:
I think we need to mention the `plugin_path` config property, but we can't always assume `/usr/lib`. How do you feel about the following?
```
To use this plugin, set `plugin_path` to the absolute path of `libirods_http_plugin-local_file.so`.

On most Linux systems, the plugin will be installed under `/usr/lib/irods_http_api`.
```

@SwooshyCueb Can we phrase this in a way that makes this cover `/usr/lib`, `/usr/lib64`, etc?

Reply:
@MartinFlores751 Let's go with the following:
```
To use this plugin, set `plugin_path` to the absolute path of `libirods_http_api_plugin-local_file.so`.

On most Linux-based systems, the HTTP API plugins will be installed under `/usr/lib/irods_http_api`.
```

We can always tweak the wording later.

Reply:
Can do.

Should we be more explicit about where `plugin_path` is, i.e. ```..., set `plugin_path` in the `user_mapping` stanza to...```?

Reply:
Yes please.

Here's an alternative.
```
Within the `user_mapping` stanza, set `plugin_path` to ...
```

Up to you on which one you use.

Reply:
Done.